vagrant/base/devstack/Vagrantfile

```diff
@@ -45,6 +46,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.network :forwarded_port, guest: 8003, host: 8003  # LMS for Bok Choy
     config.vm.network :forwarded_port, guest: 8031, host: 8031  # Studio for Bok Choy
     config.vm.network :forwarded_port, guest: 8120, host: 8120  # edX Notes Service
+    config.vm.network :forwarded_port, guest: 7474, host: 7474
```

Comment:
If it's not already, can you split this out into its own commit?
And give it a descriptive commit message.

Then I'll cherry-pick this into my PR [1].

[1] https://github.com/stvstnfrd/configuration/pull/19

Reply:
"Add port 7474" seems to be its own separate commit already.

Reply:
Cherry-picked.